volatility3/framework/plugins/windows/devicetree.py

```diff
@@ -186,6 +147,48 @@ def _generator(self) -> Iterator[Tuple]:
                 )
                 continue
 
+    @staticmethod
```

Comment:
Please don't use staticmethod inside versionable objects (even for private methods) because it prevents us being able to get back to the version from just the method.